src/rt/cover.d

```diff
@@ -24,6 +24,7 @@ private
     import core.stdc.stdio;
     import core.stdc.stdlib;
     import rt.util.utf;
+    import core.internal.parseoptions;
```

Comment:
Could be a selective import here: `: initConfigOptions`

Reply:
If we go for a selective import, we should go all the way and make it local.
Though I'm fine with keeping as it is, for consistency.

Reply:
That was exactly my take, the current style is an old style, I wasn't sure if having a mixed style is better. If it's a requirement for anyone, I can change it.

Reply:
Well, we want to gradually update the style and at least all new additions should try to avoid old-school global imports, but I can change this later today myself and then this PR should finally be good to go.